gibberish_client/src/components/Gameroom/GameroomPage.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react'
-import {Link} from 'react-router-dom'
 import PlayerListComponent from './PlayerListComponent'
 import PlayerAnswerComponent from './PlayerAnswerComponent'
 import QuestionCardComponent from './QuestionCardComponent'
 import { gamestates } from './gamestates/GameStates'
 import API from '../../api/api'
 
 const ROUND_LOADING_TIME = 3
-const ROUND_ONGOING_TIME = 10
+const ROUND_ONGOING_TIME = 25
 const ROUND_ENDED_TIME = 5
 
 class GameroomPage extends React.Component {
 
 	constructor(props) {
 		super(props)
 		this.state = {
-			roomId: props.match.params.id,
+			nickname: props.nickname,
+			roomId: props.roomId,
```

Comment:
I think it will be better if we don't set nickname and roomId as states as this creates a second source of truth. 
We can just access `nickname `and `roomId `from props directly instead when we need it :)